### ISSUE TITLE
Unable to define query parameters when adding plugin JS file

### ISSUE THREAD

**Issue by cedric-anne** [glpi-project/glpi#3843]
* Steps to reproduce (which actions have you made) :
1. Create a plugin.
2. Create a PHP script that serves JS code.
3. Register it inside plugin hooks (`$PLUGIN_HOOKS['add_javascript']['myplugin'][] = 'js/script.js.php?someparam';`).

* Expected result :
Corresponding script tag should be included inside the page. The `Html::loadJavascript()` function tests that script files exists using the provided path, without handling case where a query string is present inside this path.

* Actual result :
Script is not included.

* Your GLPI setup (you can find it in Setup > General menu, System tab) :
GLPI 9.2.3

**Comment by trasher** [comment]
Indeed, this would not work. A question: why a plugin would add a parameter along with the script path?

**Comment by cedric-anne** [comment]
Maybe it is not a good practice, but I tried to do it and, as there it just ignore the entry without raising any error, I had to debug how it works to find the cause of my problem.
My opinion is that this check of file existence is not needed. If the plugin ask loading of a non existing file, then it will just trigger a 404 error in the browser. It will have same effect for the user, but developer will be able to notice the problem in its browser tools.
If this test is keeped, adding an entry to the error log could be helpful for developers.
